contracts/Hub.sol

```diff
@@ -66,6 +72,11 @@ contract Hub {
         return true;
     }
 
+    function toggleRelayer(address _relayer) public onlyOwner {
```

Comment:
would probs prefer to be explicit here, maybe smth like:

`function updateRelayer(address relayer, bool status) public onlyOwner`

It might be theoretical / out of scope for the kind of value that we are anticipating on storing in these contracts for the pilot, and I'm not sure exactly what the attack would look like here, but these kind of non explicit interfaces are generally vulnerable to eclipse / replay attacks.

It also means we don't need the if statement, which is always nice :)